integration-tests/src/test/java/oracle/kubernetes/operator/ItModelInImage.java

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
```

Comment:
@vanajamukkara @tbarnes-us @jshum2479 @mriccell @ddsharpe @markxnelson Obviously, we've all been reviewing doc, code, and scripts, but I'm thinking that the tests need a lot more attention.  There are only two tests here (since one is commented out).  I don't think we can release without substantially more tests.  I knew that we were going to rely on some amount of manual testing -- in hindsight, that may be an incorrect choice, since there needs to be more here.

Reply:
Resolving.  Being addressed.